library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -6,6 +6,38 @@
 
 #include "internal/libspdm_common_lib.h"
 
+/**
+  Sort SPDMversion in descending order.
+
+  @param  spdm_context                A pointer to the SPDM context.
+  @param  ver_set                    A pointer to the version set.
+  @param  ver_num                    Version number.
+*/
+void spdm_secured_version_number_sort(IN OUT spdm_version_number_t *ver_set, IN uintn ver_num)
```

Comment:
Can we use same spdm_version_number_sort() ?

Reply:
Yes. I will make these two sort function same and remove the sort function to common lib.

Reply:
ok